client/src/actions/verejneActions.js

```diff
@@ -55,3 +58,30 @@ export const setCurrentPage = (newPage: number) => ({
   payload: newPage,
   reducer: () => newPage,
 })
+
+export const setAutocompleteValue = (value: string) => ({
```

Comment:
you can probably omit this, setEntitySearchFor and setEntitySearchEids and use the generic 'set value on path' action merged from profile

but I'm also fine with keeping these as they are if you decide for it